apps/src/code-studio/levels/dialogHelper.js

```diff
@@ -159,30 +187,38 @@ export function processResults(onComplete, beforeHook) {
           dialog.show();
         } else if (lastServerResponse.nextRedirect) {
           if (appOptions.dialog.shouldShowDialog) {
-            if (experiments.isEnabled('gamification') && Feedback.isLastLevel()) {
-              const stageInfo = lastServerResponse.previousStageInfo;
-              const stageName = `${window.dashboard.i18n.t('stage')} ${stageInfo.position}: ${stageInfo.name}`;
-              showDialog("success", () => {
-                const div = document.createElement('div');
-                document.body.appendChild(div);
-                const progress = Feedback.calculateStageProgress(
-                  true /* isPerfect */,
-                  0 /* hintsUsed */,
-                  appOptions.serverLevelId,
-                  false /* finiteIdealBlocks */);
-                ReactDOM.render(
-                  <StageAchievementDialog
-                    stageName={stageName}
-                    // This is a hack
-                    assetUrl={path => '/blockly/' + path}
-                    onContinue={dialogHidden}
-                    showStageProgress={experiments.isEnabled('gamification')}
-                    newStageProgress={progress.newStageProgress}
-                    numStars={Math.min(3, Math.round((progress.newStageProgress * 3) + 0.5))}
-                  />,
-                  div
-                );
-              }, () => {});
+            if (experiments.isEnabled('gamification')) {
+              const progress = Feedback.calculateStageProgress(
+                true /* isPerfect */,
+                0 /* hintsUsed */,
+                appOptions.serverLevelId,
+                false /* finiteIdealBlocks */);
+
+              // This is a hack
+              const assetUrl = path => '/blockly/' + path;
```

Comment:
I see this is moving existing code, but worth using `assetUrl` from the [page constants Redux store](https://github.com/code-dot-org/code-dot-org/blob/c9539042622b878ddfa17920f162dd7f5d9c3673/apps/src/redux/pageConstants.js#L10)?